go.mod

```diff
@@ -4,18 +4,22 @@ go 1.15
 
 require (
 	github.com/allegro/bigcache v1.2.1 // indirect
-	github.com/celo-org/celo-blockchain v1.6.1
```

Comment:
Why did we never use the 1.7 series of releases? 🤔

Reply:
unclear to me 🤷‍♀️ the dep was bumped to 1.6 after a stable 1.7 was out so I'm really not sure, possible that the 1.7 changes didn't affect the service?